wordpress-mobile/WordPress-iOS
# Reader > mark as seen: Show menu option only for posts that are followed

Ref #15355 , https://github.com/Automattic/wp-calypso/pull/49028

To test:
- Go to the Reader.
- Search for a Post or Site that you are not following.
  - Verify the card and post details options do not include the seen option.
- Search for a Post or Site that you are following.
  - Verify the card and post details options include the seen option.
- Filter `Followed` by a topic.
  - Verify the card and post details options do not include the seen option.


PR submission checklist:
